tests/test_copy.py

```diff
@@ -1274,3 +1276,58 @@ def test_copier_phase_variable(tmp_path_factory: pytest.TempPathFactory) -> None
     copier.run_copy(str(src), dst)
     assert (dst / "render").exists()
     assert (dst / "render").read_text() == "render"
+
+
+def test_relative_render_path_outside_destination_raises_error(
+    tmp_path_factory: pytest.TempPathFactory,
+) -> None:
+    root = tmp_path_factory.mktemp("root")
+    (src := root / "src").mkdir()
+    (dst := root / "dst").mkdir()
+    build_file_tree(
+        {
+            root / "forbidden.txt": "foo",
+            src / "{{ pathjoin('..', 'forbidden.txt') }}": "bar",
```

Comment:
This is the problematic like for Windows right? Because of the single-quotes in the filename?

Reply:
No, those are fine (I tried on Windows). The current error seems to be caused by the colon in the Windows drive part (`C:`) which isn't allowed in a directory/file name. I think moving this to, e.g., a computed value and using the variable in the Jinja expression might work. I'll try to figure it out over the weekend.